src/app/CMakeLists.txt

```diff
@@ -273,6 +273,20 @@ if (OS_IS_WIN)
               USE_SOURCE_PERMISSIONS
           )
 
+    if (MSVC AND DEVELOPER)
```

Comment:
DEVELOPER - I think we need to think of another name, it’s not clear what it is and where it comes from 
I think we need to check the name of the cmake generator 
Or something like this: VS_CMAKE_GENERATOR

Reply:
I see that this is also used for VSCode, then we still need to think about a good name.

Reply:
I just realize that the `MSVC` check is sufficient for this part of the configuration (configuring the debugging command correctly). This then leaves the `DEVELOPER` flag to adding the post-build step to copy the executable to the install directory, which we can still discuss if we want this at all.

Reply:
If we decide to keep the copy step, let's name the flag something like: `OPTIMIZED_DEPLOY`

Reply:
Sorry, no, it still is needed for Visual Studio, which doesn't have pre-launch tasks AFAIK.

Reply:
... but then it's not so much a matter of optimizing deployment anymore. `COPY_TO_INSTALL` ? Doesn't say everything but it gives a good clue.

Reply:
... or maybe we can just add that post-build step just if `MSVC`. Won't hurt CI anyway, as it uses Ninja. Then no need for that extra flag.

Reply:
`COPY_TO_INSTALL` - acceptable for me